src/D2L.CodeStyle.Analyzers/Immutability/ImmutableDefinitionChecker.cs

```diff
@@ -330,35 +349,28 @@ ISymbol memberSymbol
 		/// <param name="memberTypeSyntax">The syntax for the type of the field/property</param>
 		/// <param name="assignment">The assignment syntax for the field/property (possibly null)</param>
 		/// <returns>null if no checks are needed, otherwise a bunch of stuff.</returns>
-		private (ITypeSymbol, ImmutableTypeKind, Func<Location>)? GetStuffToCheckForMember(
-			ITypeSymbol memberType,
-			TypeSyntax memberTypeSyntax,
+		private (ITypeSymbol, ImmutableTypeKind, Func<Location>)? GetStuffToCheckForAssignment(
 			ExpressionSyntax assignment

```

Comment:
We lost the arguments for `memberType` and `memberTypeSyntax`. This is good because this function is just doing stuff for a particular assignment, and it makes sense that it doesn't care about the member's type at all.

Reply:
As we talked about awhile ago, we no longer need to return `Func<Location>` because its not deferring any work anymore, but I'll leave that for a follow-up